php/tests/generated_class_test.php

```diff
@@ -12,10 +12,13 @@
 use Foo\TestIncludeNamespaceMessage;
 use Foo\TestIncludePrefixMessage;
 use Foo\TestMessage;
+use Foo\TestMessage\Sub;
 use Foo\TestMessage_Sub;
```

Comment:
Do we need the old one?

Reply:
We run a test with it [here](https://github.com/google/protobuf/pull/4536/files#diff-71f0202385891cc441f357b417daa431R407) to verify compatibility. So while it's not needed (we could just reference it as `Foo\TestMessage_Sub` in the typehint), it's cleaner to import it.

Reply:
I prefer to test it in the compatibility test. I will set it up. You don't need to add compatibility test in the code.

Reply:
This is a very important tests that I'd like to make sure gets integrated properly. I can move them to the compatibility test if you'd like, but I do think these tests need to be a part of this PR.

Reply:
ok, just leave it here for now.